client/www/pages/docs/index.md

```diff
@@ -257,3 +257,11 @@ export default App
 Go to `localhost:3000` and follow the final instruction to load the app!
 
 Huzzah 🎉 You've got your first Instant web app running! Check out the [**Explore**](/docs/init) section to learn more about how to use Instant :)
+
+{% callout type="note" %}
+
+**Level up your app!**
```

Comment:
I don't think we should include this yet in our Getting Started. This is a more advanced concept, I think it would be better to recommend folks to go deeper and check out our `init` docs